internal/units/units.go

```diff
@@ -40,6 +46,15 @@ func BytesStringBase2(b int64) string {
 	return toDecimalUnitString(float64(b), 1024.0, base2UnitPrefixes, "B")
 }
 
+func BytesString(b int64) string {
+	if v := os.Getenv(bytesStringBase2Envar); v != "" {
+		if bool, err := strconv.ParseBool(v); err == nil && bool {
+			return BytesStringBase2(b)
+		}
+	}
```

Comment:
Think it's extraneous to check for a non-empty value as we're parsing a bool anyways:
```suggestion
func BytesString(b int64) string {
	if bool, err := strconv.ParseBool(os.Getenv(bytesStringBase2Envar)); err == nil && bool {
		return BytesStringBase2(b)
	}
```